### PR TITLE
DOC-677 | 3.12.0 release

### PR DESCRIPTION
**Pull request by Simran-B**
### Description

- KI for inverted index progress
- Define what hidden indexes are
- collation Analyzer breaking change
- CircleCI: releases should be compiled from release instead of version branch

#### Upstream PRs

- 3.10: 
- 3.11: 
- 3.12: 
- 4.0:

### CODE REVIEW

File: site/content/3.12/release-notes/version-3.12/incompatible-changes-in-3-12.md [hunk 1]
```diff
@@ -170,6 +170,27 @@ onward and will be removed in a future version.
 You can use [Stream Transactions](../../develop/transactions/stream-transactions.md)
 instead in most cases, and in some cases AQL can be sufficient.
 
+## Breaking changes to the `collation` Analyzer
+
+The [`collation` Analyzer](../../index-and-search/analyzers.md#collation) lets
+you adhere to the alphabetic order of a language in range queries. For example,
+using a Swedish locale (`sv`), the sorting order is `å` after `z`, whereas using
```
Comment (MBkkt) [inline site/content/3.12/release-notes/version-3.12/incompatible-changes-in-3-12.md:177]:
```suggestion
using a Swedish locale (`sv`), the sorting order is `å` after `a`, whereas using
```

File: site/content/4.0/release-notes/version-3.12/incompatible-changes-in-3-12.md [hunk 2]
```diff
@@ -170,6 +170,27 @@ onward and will be removed in a future version.
 You can use [Stream Transactions](../../develop/transactions/stream-transactions.md)
 instead in most cases, and in some cases AQL can be sufficient.
 
+## Breaking changes to the `collation` Analyzer
+
+The [`collation` Analyzer](../../index-and-search/analyzers.md#collation) lets
+you adhere to the alphabetic order of a language in range queries. For example,
+using a Swedish locale (`sv`), the sorting order is `å` after `z`, whereas using
```
Comment (MBkkt) [inline site/content/4.0/release-notes/version-3.12/incompatible-changes-in-3-12.md:177]: same

File: site/content/4.0/release-notes/version-3.12/incompatible-changes-in-3-12.md [hunk 3]
```diff
@@ -170,6 +170,27 @@ onward and will be removed in a future version.
...
+ArangoDB 3.12 bundles an upgraded version of the ICU library. It is used for
+Unicode character handling including text sorting. Because of changes in ICU,
+data produced by the `collation` Analyzer in previous versions is not compatible
+with ArangoDB v3.12. You need to **recreate inverted indexes and Views that use
+`collation` Analyzers** to ensure that they work correctly. Otherwise,
+range queries involving the `collation` Analyzers and indexes created in v3.11
+or older versions may behave in unpredicted ways.
+
+Note that sorting by the output of the `collation` Analyzer like
+`SORT TOKENS(<text>, <collationAnalyzer>` is still not a supported feature and
```
Comment (MBkkt) [inline site/content/4.0/release-notes/version-3.12/incompatible-changes-in-3-12.md:191]:
```suggestion
`SORT TOKENS(<text>, <collationAnalyzer>)` is still not a supported feature and
```

File: site/content/3.12/release-notes/version-3.12/incompatible-changes-in-3-12.md [hunk 4]
```diff
@@ -170,6 +170,27 @@ onward and will be removed in a future version.
...
+ArangoDB 3.12 bundles an upgraded version of the ICU library. It is used for
+Unicode character handling including text sorting. Because of changes in ICU,
+data produced by the `collation` Analyzer in previous versions is not compatible
+with ArangoDB v3.12. You need to **recreate inverted indexes and Views that use
+`collation` Analyzers** to ensure that they work correctly. Otherwise,
+range queries involving the `collation` Analyzers and indexes created in v3.11
+or older versions may behave in unpredicted ways.
+
+Note that sorting by the output of the `collation` Analyzer like
+`SORT TOKENS(<text>, <collationAnalyzer>` is still not a supported feature and
```
Comment (MBkkt) [inline site/content/3.12/release-notes/version-3.12/incompatible-changes-in-3-12.md:191]:
```suggestion
`SORT TOKENS(<text>, <collationAnalyzer>)` is still not a supported feature and
```

**Review by MBkkt — APPROVED** [review]

File: site/content/3.12/release-notes/version-3.12/incompatible-changes-in-3-12.md (hunk 1 above)
Comment (Simran-B) [reply site/content/3.12/release-notes/version-3.12/incompatible-changes-in-3-12.md:177]: @MBkkt It's actually abc...xyzåäö, see https://en.wikipedia.org/wiki/Swedish_alphabet

Comment (MBkkt) [reply site/content/3.12/release-notes/version-3.12/incompatible-changes-in-3-12.md:177]: ok, but your example isn't clear, you can use same letters for different locales

**Review by ansoboleva — APPROVED** [review]

**Review by KVS85 — APPROVED** [review]

**Pull request merged**